internal/pkg/cli/init.go

```diff
@@ -78,21 +72,28 @@ func NewInitOpts() (*InitOpts, error) {
 		manifestWriter: ws,
 		prompt:         prompt,
 	}
+	initEnv := &InitEnvOpts{
+		EnvName:       defaultEnvironmentName,
+		EnvProfile:    "default",
+		IsProduction:  false,
+		envCreator:    ssm,
+		projectGetter: ssm,
+		envDeployer:   cloudformation.New(sess),
```

Comment:
How do you envision we will surface this environment creation configuration to the user? The environment name, profile, prod flag, and deployer session could all be variable values. There's also the `loadEnv` function below which seems to exist because we don't have the project name in context at the time of creating this structure which makes me feel like the creation of the `InitEnvOpts` struct should move to a place where we have definitive inputs. Thoughts?

Reply:
> How do you envision we will surface this environment creation configuration to the user? The environment name, profile, prod flag, and deployer session could all be variable values.

Good question, I was punting on this for now. We'll prob create flags for them in `init` just like we do for `dockerfile`.

> There's also the loadEnv function below which seems to exist because we don't have the project name in context at the time of creating this structure which makes me feel like the creation of the InitEnvOpts struct should move to a place where we have definitive inputs. Thoughts?

Totally agree, it feels pretty awkward now :/  
I've changed the subcommands to refer to the project name strictly with `viper.GetString(projectFlag)`. This way we can remove this janky  injection :)